### DISCUSSION TITLE
Question: "fn" keyword

### DISCUSSION THREAD

**Discussion by renlite** [c3lang/c3c#1181] [General]
Why do you use the "fn" keyword for functions and methods. It could be simply omitted and would fit the C declaration.
What is(are) the benefit(s) using the "fn" keyword? It's more to wrtie and seems not to help with readability of code.

**Comment by lerno** [comment]
I've tried to answer that question here: https://c3.handmade.network/blog/p/8886-why_does_c3_use_%2527fn%2527
